presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraVersion.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cassandra;
+
+import static java.util.Objects.requireNonNull;
+
+public class CassandraVersion
```

Comment:
In this case, I have to pass Cassandra Version directly to and compare using below logic.
`if(VersionNumber.parse(<actualversion>).compareTo(VersionNumber.parse("2.2.0")) == -1)
{
//Then version is less than 2.2.
}`

Kindly confirm.